go/configure/action.yml

```diff
@@ -1,10 +1,7 @@
-name: 'go-setup'
+name: 'go-configure'
```

Comment:
Renamed to reduce confusion with the old action since this no longer installs go.